### ISSUE TITLE
double meaning of index k

### ISSUE THREAD

**Issue by rmnldwg** [YoelPH/mixture-model-paper#2]
In equation 5, you enumerate the patients using the index $k$. However, you then also use $k$ as subscript to the state vector: $\xi_k$. But the subset to this vector should denote which of the $2^5$ states the patient $k$ is in, no? Put differently, patient number $k=13$ is not necessarily in state $\xi_{13}$.

**Comment by YoelPH** [comment]
I agree. My idea is to reindex the $\xi$ to represent the patient specific involvement instead of one of the numbered states. This makes it easier to introduce the mixture model.

**Comment by rmnldwg** [comment]
I think we originally distinguished between the enumerated states $\xi_i$ and any patient's involvement $x_k$. Then one can express that patient $k$ is in state $i$: $x_k=\xi_i$.

**Comment by YoelPH** [comment]
I see, so I'd rather change it to $x_k$ I guess?

**Comment by rmnldwg** [comment]
It would make the notation consistent with the previous papers. But we can think of another way to put it as well 🤷🏻
